resources/lang/en/general.php

```diff
@@ -46,6 +46,12 @@
     'now'                 => 'Now',
     'vote_with'           => 'Vote With',
 
+    'arkconnect' => [
+        'mainnet_network' => 'You\'re viewing data from the main network, but your wallet is connected to test network (ARK Testnet). To use ARKScan, please switch to <span class="font-semibold">test.arkscan.io</span>.',
+        'testnet_network' => 'You\'re viewing data from the test network, but your wallet is connected to main network (ARK Mainnet). To use ARKScan, please switch to <span class="font-semibold">live.arkscan.io</span>.',
+        'wrong_network'   => 'You\'re unable to do this because you\'re connected to the wrong network.',
```

Comment:
maybe along the lines of "Wrong network, please switch to [network] to enable this action." ?